server/sonar-server/src/test/java/org/sonar/server/edition/StandaloneEditionManagementStateImplTest.java

```diff
@@ -277,6 +278,91 @@ public void startAutomaticInstall_fails_with_ISE_if_called_after_manualInstall()
   }
 
   @Test
+  public void uninstall_fails_with_ISE_if_not_started() {
+    expectISENotStarted();
+
+    underTest.uninstall();
+  }
+
+  @Test
+  public void uninstall_sets_resets_fields_after_start() {
```

Comment:
"_sets" and "_resets", one if too much :)

Reply:
ok